src/structures/User.js

```diff
@@ -75,6 +75,28 @@ class User extends Base {
       this.avatar = null;
     }
 
+    if ('banner' in data) {
+      /**
+       * The user banner's hash
+       * <info>The user must be force fetched</info>
+       * @type {?string}
+       */
+      this.banner = data.banner;
+    } else if (typeof this.banner !== 'string') {
+      this.banner = null;
+    }
+
+    if ('accent_color' in data) {
+      /**
+       * The base 10 accent color of the user
+       * <info>The user must be force fetched</info>
+       * @type {?number}
+       */
+      this.accentColor = data.accent_color;
+    } else if (typeof this.accentColor === 'undefined') {
+      this.accentColor = null;
+    }
```

Comment:
Considering banner_color might be removed, why don't we rename this property to bannerColor? I feel like "accent color of the user" can be quite vague

Reply:
We align with the names of the API properties, which is accent_color

Reply:
also it's "profile color" in the ui, not banner color

ig the description could be improved tho (d.js's property descriptions are generally pretty undescriptive)

Reply:
Seems like it's only called that because of the old deprecated banner_color property. If you really insist in keeping this name then the documentation needs to be improved because it makes no sense as it is at the moment

Reply:
```suggestion
    if ('accent_color' in data) {
      /**
       * The base 10 color of the user's banner
       * <info>The user must be force fetched for this property to be present</info>
       * @type {?number}
       */
      this.accentColor = data.accent_color;
    } else if (typeof this.accentColor === 'undefined') {
      this.accentColor = null;
    }
```

Reply:
its not really the "color of the user's banner" though, it's a separate setting from the banner and it isn't locked to Nitro

![image](https://user-images.githubusercontent.com/11778454/129777545-5916b1bc-cf49-4d2c-b40c-6176b1a4b859.png)

Reply:
I said that because it's the color that stays in the place where the banner would otherwise be so it's sort of the color of the banner and that explanation would be clear enough imo